library/Icingadb/Redis/VolatileState.php

```diff
@@ -17,15 +17,15 @@ class VolatileState
 
     /** @var array Set of keys to sync */
     public static $keys = [
-        'attempt',
```

Comment:
In sight of #566, this change and the one below should be reverted here, to avoid a merge conflict.